ios/RNSScreen.mm

```diff
@@ -61,8 +61,7 @@ - (instancetype)initWithFrame:(CGRect)frame
   }
   return self;
 }
-#endif // RCT_NEW_ARCH_ENABLED
-
+#else
 - (instancetype)initWithBridge:(RCTBridge *)bridge
```

Comment:
I think we should also check if is new architecture enabled in places where we're declaring initWithBridge (e.g. RNSFullWindowOverlay, RNSSearchBar). I also see some inconsistency here - in RNSScreenStackHeaderSubview we're declaring this initWithBridge into the `#pragma mark - Paper specific` declaration. Maybe we should move this initializer into the pragmas everywhere? What do you think?

Reply:
Let's leave that on a separate PR 👍

Reply:
Yeah, we can do it in some follow-up for sure. It does not change anything on new arch since this method is just never called.